syntax = "proto3";
package pkg;





/*
DeletePackage请求
*/
message DeletePackageRequest {
    /*
    包Id
    */
    string packageId = 1 ;
}

/*
DeletePackage返回
*/
message DeletePackageResponse {
    /*
    包Id
    */
    string packageId = 1 ;
}

/*
DeletePackageApi返回
*/
message DeletePackageResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    pkg.DeletePackageResponse data = 4 ;
}

